src/components/personLink/PersonLink.tsx

```diff
@@ -0,0 +1,65 @@
+import { useContext } from 'react';
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PeopleList } from '../../PeopleContext';
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.